### PR TITLE
Log conntrack operations

### PR DESCRIPTION
**Pull request by aojea**
/kind cleanup
/kind flake

```release-note
UDP and SCTP protocols can left stale connections that need to be cleared to avoid services disruption, but they can cause problems that are hard to debug.
Kubernetes components using a loglevel greater or equal than 4 will log the conntrack operations and its output, to show the entries that were deleted.
```

xref: https://github.com/kubernetes/kubernetes/issues/91236

### CODE REVIEW

**Comment by aojea** [comment]
/assign @wojtek-t 
let see if we are deleting more entries than needed or we are racing, causing
https://github.com/kubernetes/kubernetes/issues/91236

File: pkg/proxy/iptables/proxier.go [hunk 1]
```diff
@@ -1625,11 +1625,13 @@ func (proxier *Proxier) syncProxyRules() {
 
 	// Finish housekeeping.
 	// TODO: these could be made more consistent.
+	klog.V(4).Infof("Deleting stale services IPs: %v", staleServices.UnsortedList())
```
Comment (aojea) [inline pkg/proxy/iptables/proxier.go:1628]: is this the right way to log this , %v?

File: pkg/proxy/iptables/proxier.go [hunk 2]
```diff
@@ -1625,11 +1625,13 @@ func (proxier *Proxier) syncProxyRules() {
...
 	for _, svcIP := range staleServices.UnsortedList() {
 		if err := conntrack.ClearEntriesForIP(proxier.exec, svcIP, v1.ProtocolUDP); err != nil {
 			klog.Errorf("Failed to delete stale service IP %s connections, error: %v", svcIP, err)
 		}
 	}
+	klog.V(4).Infof("Deleting stale endpoint connections: %v", endpointUpdateResult.StaleEndpoints)
```
Comment (aojea) [inline pkg/proxy/iptables/proxier.go:1634]: same question as above, is this the correct way to do it?

File: pkg/proxy/iptables/proxier.go (hunk 1 above)
Comment (aojea) [reply pkg/proxy/iptables/proxier.go:1628]: or is better to add a log entry by IP inside the for loop?

Comment (wojtek-t) [reply pkg/proxy/iptables/proxier.go:1628]: "%v" seem to work fine for map[string]Empty{}

**Comment by wojtek-t** [comment]
/lgtm
/approve

[We're not using --v=4 verbosity in 5k tests, so that shouldn't affect us at scale.]

**Pull request merged**